include/groonga/column.h

```diff
@@ -149,6 +149,35 @@ typedef struct _grn_column_cache grn_column_cache;
 #define GRN_COLUMN_NAME_MEAN     "_mean"
 #define GRN_COLUMN_NAME_MEAN_LEN (sizeof(GRN_COLUMN_NAME_MEAN) - 1)
 
+/**
+ * \brief Create a new column in a table.
+ *
+ * \param ctx The context object.
+ * \param table The target table.
+ * \param name The name of the column. The name must be unique within the table.
+ *             Duplicate column names are not allowed in the same table.
+ * \param name_size The size of the `name` parameter in bytes.
+ * \param path The file path where the column will be stored. This parameter is
+ *             only effective if the \ref GRN_OBJ_PERSISTENT flag is specified
+ *             in `flags`.
+ * \param flags Available values:
+ *   * \ref GRN_OBJ_PERSISTENT
+ *   * \ref GRN_OBJ_COLUMN_INDEX
+ *   * \ref GRN_OBJ_COLUMN_SCALAR
+ *   * \ref GRN_OBJ_COLUMN_VECTOR
+ *   * \ref GRN_OBJ_COMPRESS_ZLIB
+ *   * \ref GRN_OBJ_COMPRESS_LZ4
+ *   * \ref GRN_OBJ_COMPRESS_ZSTD
+ *   * When only used with \ref GRN_OBJ_COLUMN_INDEX as follows:
```

Comment:
ここで箇条書きを分けることってできますか？

```suggestion
 *
 *   When only used with \ref GRN_OBJ_COLUMN_INDEX as follows:
 *
```

Reply:
fix: [8559e5c](https://github.com/groonga/groonga/pull/2070/commits/8559e5ce08ec56a1521f05c3b13db1521747acf4) 箇条書きを分けられなかったので、箇条書きをネストしてみました。

![Screenshot from 2024-11-11 14-53-45](https://github.com/user-attachments/assets/08314ddd-89db-496c-964a-46b70df3051b)